### PR TITLE
AP_Soaring: Correct metadata formatting

### PR DESCRIPTION
**Pull request by Hwurzburg**

### CODE REVIEW

**Comment by Hwurzburg** [comment]
This should do it...

**Review by tridge — APPROVED** [review]

**Comment by magicrub** [comment]
Let's have @samuelctabor take a quick peek before we merge

**Comment by magicrub** [comment]
Also, the commit and the PR title say this is AP_Airspeed which it isn't

File: libraries/AP_Soaring/AP_Soaring.cpp [hunk 1]
```diff
@@ -142,16 +142,16 @@ const AP_Param::GroupInfo SoaringController::var_info[] = {
     // 19 reserved for POLAR_LEARN.
 
     // @Param: THML_ARSPD
-    // @DisplayName: Specific setting for airspeed when thermalling.
-    // @Description: If non-zero this airspeed will be used when thermalling.
-    // @Values: 0: Use TRIM_ARSPD_CM, >0: Target airspeed in m/s
+    // @DisplayName: Specific setting for airspeed when in a thermal in SOARING.
```
Comment (samuelctabor) [inline libraries/AP_Soaring/AP_Soaring.cpp:145]: suggest " @DisplayName: Specific setting for airspeed when soaring in THERMAL mode"

File: libraries/AP_Soaring/AP_Soaring.cpp [hunk 2]
```diff
@@ -142,16 +142,16 @@ const AP_Param::GroupInfo SoaringController::var_info[] = {
...
+    // @Description: If non-zero this airspeed will be used when thermalling. A value of 0 will use TRIM_ARSPD_CM.
+    // @Range: 0 50
     // @User: Advanced
     AP_GROUPINFO("THML_ARSPD", 20, SoaringController, soar_thermal_airspeed, 0),
 
     // @Param: CRSE_ARSPD
-    // @DisplayName: Specific setting for airspeed when cruising in AUTO.
-    // @Description: If non-zero this airspeed will be used when cruising in AUTO. If set to -1, airspeed will be selected based on speed-to-fly theory.
-    // @Values: -1: Use speed-to-fly computed value, 0: Use TRIM_ARSPD_CM, >0: Use this value in m/s instead of TRIM_ARSPD_CM.
+    // @DisplayName: Specific setting for airspeed when cruising while SOARING.
```
Comment (samuelctabor) [inline libraries/AP_Soaring/AP_Soaring.cpp:152]: Suggest "    // @DisplayName: Specific setting for airspeed when soaring in AUTO mode"

File: libraries/AP_Soaring/AP_Soaring.cpp [hunk 3]
```diff
@@ -142,16 +142,16 @@ const AP_Param::GroupInfo SoaringController::var_info[] = {
...
+    // @Description: If non-zero this airspeed will be used when cruising between thermals. If set to -1, airspeed will be selected based on speed-to-fly theory. If set to 0, then TRIM_ARSPD_CM will be used while cruising between thermals.
```
Comment (samuelctabor) [inline libraries/AP_Soaring/AP_Soaring.cpp:153]: Suggest " // @Description: If non-zero this airspeed will be used when cruising between thermals in AUTO mode"

**Review by samuelctabor — CHANGES_REQUESTED** [review]

File: libraries/AP_Soaring/AP_Soaring.cpp (hunk 1 above)
Comment (Hwurzburg) [reply libraries/AP_Soaring/AP_Soaring.cpp:145]: done

File: libraries/AP_Soaring/AP_Soaring.cpp (hunk 3 above)
Comment (Hwurzburg) [reply libraries/AP_Soaring/AP_Soaring.cpp:153]: Done

File: libraries/AP_Soaring/AP_Soaring.cpp (hunk 2 above)
Comment (Hwurzburg) [reply libraries/AP_Soaring/AP_Soaring.cpp]: Done

**Comment by samuelctabor** [comment]
Looks good thanks Henry

**Pull request merged**